### DISCUSSION TITLE
HAos RaspberryMatic Addon findet keine Geräte mehr / kein Pairing möglich nach HA Reboot

### DISCUSSION THREAD

**Discussion by ElGauchooooo** [OpenCCU/OpenCCU#1963] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

System sieht wie folgt aus:

- RPI 4 / 8 GB
- Home Assistant Operating System 8.2
- HM-MOD-RPI-PCB 

habe heute RaspberryMatic installiert und je zwei Thermostate hinzugefügt (HmIP-eTRV-B) sowie zwei Fenstersensoren (HmIP-SWDM). Wurden beide problemlos erkannt und liesen sich steuern / auslesen. Habe danach Home Assistant neu gestartet, Add-On läuft wieder normal an und die Logs sehen auch normal aus:

```
Identifying Homematic RF-Hardware: ....HmRF: HM-MOD-RPI-PCB/GPIO@fe201000.serial, HmIP: HM-MOD-RPI-PCB/GPIO@fe201000.serial, OK
Updating Homematic RF-Hardware: HM-MOD-RPI-PCB: 2.8.6, not necessary, OK
```

Allerdings kann ich nun keine neuen Geräte pairen (mit 3 weiteren Thermostaten getestet + 2 Fenstersensoren), desweiteren schlägt die Kommunikation mit den bestehenden Sensoren / Aktoren fehl. Unter System -> Devices werden sie als `UNREACH` aufgelistet, im Device-Tab wird bei einer Änderung das Gerät als `Device communication disturbed` gelistet. (einer der Aktoren ist ca 2m vom PI entfernt).

Was könnte diese Verhalten verursachen?

**Comment by jens-maus** [comment, marked as answer]
> * RPI 4 / 8 GB

Das ist der Verursacher! Ein RaspberryPi4 ist leider bekannt dafür ein extremer Funkstörer in seiner näheren Umgebung zu sein. Gerade wenn etwas an den USB3 Ports angeschlossen ist, aber auch ohne das das der Fall ist kommt es zu massiven Funkbeeinträchtigungen bei Einsatz eines RaspberryPi4. Siehe dazu auch:

https://homematic-forum.de/forum/viewtopic.php?f=31&t=65293
